build_tools/azure/render_meta.py

```diff
@@ -0,0 +1,58 @@
+import os
+import sys
+
+from jinja2 import Environment, FileSystemLoader
+from pathlib import Path
+
+# Since conda is only on Azure Pipelines, we can use their env variables
+ROOT_DIRECTORY = Path(os.getenv('BUILD_SOURCESDIRECTORY'))
+DIST_PATH = ROOT_DIRECTORY / 'dist'
+VERSION_FILE = ROOT_DIRECTORY / 'pmdarima' / 'VERSION'
+REQUIREMENTS_FILE = ROOT_DIRECTORY / 'requirements.txt'
+
+# conda is weird about yml vs yaml, so we have to use yaml
+OUTPUT_DIR = ROOT_DIRECTORY / 'conda'
+OUTPUT_FILE = OUTPUT_DIR / 'meta.yaml'
+
+TEMPLATE_PATH = Path(__file__).parent
+TEMPLATE_ENVIRONMENT = Environment(
+    autoescape=False,
+    loader=FileSystemLoader(str(TEMPLATE_PATH.resolve())),
+    trim_blocks=False
+)
+
+# Find the version
+try:
+    VERSION = open(str(VERSION_FILE.resolve())).readline().strip()
+except FileNotFoundError:
+    VERSION = '0.0.0'
```

Comment:
Can/should we raise here? I would really hope that come Conda release time we have a `VERSION` file :)

Reply:
I agree we shouldn't release if we can't find the VERSION file, however I don't think it can happen here. Conda's `meta.yaml` file is required to build the package, and `version` is a required key in that file. So this is required to build the conda distribution even if we have no intention of distributing it. I can add something similar to what I did [here](https://github.com/alkaline-ml/pmdarima/pull/263#discussion_r357141289) on Github Actions

Reply:
This is handled as of 70cdd4f. See [here](https://dev.azure.com/tgsmith61591gh/pmdarima/_build/results?buildId=1415&view=logs&j=2e703997-b85c-5955-fe15-cdf93bef6757&t=e51ec69b-f0fa-593a-ddf4-8cce6a79d81c&l=2)